### PR TITLE
feat(itemreg): replace serial with IMEI in phone registration

### LINKED ISSUE #1054: Ask for an IMEI number instead of a serial number for phone registrations
## Feature description
Instead of asking for a serial number when registering a phone, ask for an IMEI number and provide instructions on how to find it.

## Rationale for adding feature
* **Serial numbers aren't printed on the outside of devices anymore.** [Apple only prints the IMEI on the outside of their phones](https://support.apple.com/en-us/HT204073#iPhone), requiring an unlock to find the serial number, which is useless if someone finds a phone but can't unlock it to find the serial number. Samsung only prints the IMEI number as well, but their website doesn't mention it so you'll just have to take my word for it. [Google Pixels appear to only have the IMEI on the back](https://support.google.com/store/answer/3333000?hl=en).
* **People don't know how to find their phone's serial number.** Looking through the current list of registered phones, some serial numbers are "idk".

## Suggested implementation(s) (if applicable)
- Change the form to ask for an IMEI number.
- Add instructions on how to find said IMEI number on both the registration page and the "lookup found device" page:
  - Dial `*#06#` on the dialer
  - Look on the back of the phone or in the SIM card tray as detailed on manufacturer's websites
  - Look in phone settings

### PR DESCRIPTION
**Pull request by nkenschaft**
closes #1054

### CODE REVIEW

File: intranet/templates/itemreg/register_form.html [hunk 1]
```diff
@@ -89,9 +89,20 @@ <h2>
...
     </table>
 
     </form>
+    {% if type == "phone" %}
+    <br>
+    <h2>How to find your phone's IMEI</h2>
+    <p>The IMEI can generally be found by dialing <code>*#06#</code>.</p>
+    <p>For manufacturer specific instructions, see the following links:</p>
+    <ul>
+        <li><a href=\"https://support.apple.com/en-us/HT204073\">Apple</a></li>
```
Comment (Laur04) [inline intranet/templates/itemreg/register_form.html:102]:
```suggestion
        <li><a href="https://support.apple.com/en-us/HT204073">Apple</a></li>
```

File: intranet/templates/itemreg/register_form.html [hunk 2]
```diff
@@ -89,9 +89,20 @@ <h2>
...
+        <li><a href=\"https://support.google.com/store/answer/3333000?hl=en\">Google</a></li>
```
Comment (Laur04) [inline intranet/templates/itemreg/register_form.html:103]:
```suggestion
        <li><a href="https://support.google.com/store/answer/3333000?hl=en">Google</a></li>
```

File: intranet/templates/itemreg/register_form.html [hunk 3]
```diff
@@ -89,9 +89,20 @@ <h2>
...
+        <li><a href=\"https://www.samsung.com/us/support/answer/ANS00048604/\">Samsung</a></li>
```
Comment (Laur04) [inline intranet/templates/itemreg/register_form.html:104]:
```suggestion
        <li><a href="https://www.samsung.com/us/support/answer/ANS00048604/">Samsung</a></li>
```

File: intranet/templates/itemreg/register_form.html [hunk 4]
```diff
@@ -89,9 +89,20 @@ <h2>
...
     {% csrf_token %}
     {{ form.as_table }}
-    <tr><td>&nbsp;</td><td><br><input type="submit"></td></tr>
+    <tr><td>&nbsp;</td><td><br><input type="submit" value="Register Device"></td></tr>
     </table>
 
     </form>
+    {% if type == "phone" %}
+    <br>
+    <h2>How to find your phone's IMEI</h2>
```
Comment (Laur04) [inline intranet/templates/itemreg/register_form.html:98]:
```suggestion
    <h2>Find your IMEI:</h2>
```

**Review by Laur04 — CHANGES_REQUESTED** [review]
Otherwise LGTM

**Comment by nkenschaft** [comment]
> Otherwise LGTM
> 
> Edit: Also, you need to run `scripts/build_sources.sh` for checks to pass.

Done and included suggestions.

File: intranet/templates/itemreg/register_form.html [hunk 5]
```diff
@@ -89,9 +89,18 @@ <h2>
     <table>
     {% csrf_token %}
     {{ form.as_table }}
-    <tr><td>&nbsp;</td><td><br><input type="submit"></td></tr>
+    <tr><td>&nbsp;</td><td><br><input type="submit" value="Register Device"></td></tr>
     </table>
 
     </form>
-</div>
-{% endblock %}
```
Comment (Laur04) [inline intranet/templates/itemreg/register_form.html:97]: This shouldn't be removed. It's causing your checks to fail.
```
</div>
{% endblock %}
```

**Review by Laur04 — CHANGES_REQUESTED** [review]

File: intranet/templates/itemreg/register_form.html (hunk 5 above)
Comment (nkenschaft) [reply intranet/templates/itemreg/register_form.html:97]: > This shouldn't be removed. It's causing your checks to fail.

Whoops, I don't know how I removed that. It's fixed now.

**Review by Laur04 — APPROVED** [review]
LGTM

**Review by etnguyen03 — APPROVED** [review]

**Pull request merged**